Daviddevbranch

ChangerEX opened the pull request:
added level

ChangerEX committed:
a difficult level by David

ChangerEX committed:
Merge branch 'premain' into daviddevbranch

adenprince commented on Assets/2DGamekit/Art/TileMapPalettes/Tileset/TilesetRockWaterBlockers.prefab:
```diff
@@ -43,67 +43,74 @@ Tilemap:
       m_TileSpriteIndex: 0
       m_TileMatrixIndex: 0
       m_TileColorIndex: 0
-      m_ObjectToInstantiate: {fileID: 0}
```
I'm planning to revert the changes to `TilesetGamekit.prefab` and `TilesetRockWaterBlockers.prefab` unless there's a reason they were changed.

adenprince commented:
It seems like changing the player invulnerability duration to 1.25 is causing weird interaction with the acid and soft locks. I'm planning to revert that change.

adenprince commented on Assets/Sprites/invis.asset:
```diff
@@ -0,0 +1,36 @@
+%YAML 1.1
```
It seems like `Square.png` and `invis.asset` are intended to be used for invisible hitboxes. `Square.png` is made invisible in the scene, and `invis.asset` seems to be unused. I might simplify how invisible hitboxes are set up in the level and remove these files.

adenprince committed:
Revert tileset changes

adenprince committed:
Update davidLevel.unity and revert unneeded changes

Changes in davidLevel.unity:
- Remove sprite renderer from invisible hitboxes
- Rename invisible hitboxes
- Remove random floating invisible hitbox
- Revert player invulnerability duration change
Additional changes:
- Remove Square.png
- Remove invis.asset
- Revert TestScene2.unity deletion

adenprince committed:
Revert TestScene2.unity.meta changes

adenprince commented:
I made all the changes that I described in previous comments.

Zlrobbins9 committed:
Merge branch 'premain' into daviddevbranch
